### DISCUSSION TITLE
Step 9) luk sag

### DISCUSSION THREAD

**Discussion by rebeccakol** [Kortforsyningen/FIRE#326] [Q&A] [answered]
Har forsøgt at teste "luk sag" men får denne besked. Kan ikke lige gennemskue hvad det betyder?

(fire) F:\GRF\Data\GEO\BC\Niv_Opgaver\2020\5D_KDI_KALVEHAVE>fire niv luk-sag 5D_KDI_KALVEHAVE

5D_KDI_KALVEHAVE er kan ikke indsættes i test-databasen, da det er oprettet i Series([], )-databasen!

**Comment by kbevers** [comment, marked as answer]
Den kommer fordi projektfilen er lavet med forrige version af fire (1.0.0-alpha1). Du kan omgå problemet ved at indsætte en ny linje på fanebladet "Parametre", hvor der i kolonnen "navn" står "Database" og i kolonnen værdi står "test". Det burde være tilstrækkeligt for at kunne komme videre.

**Reply by rebeccakol** [reply]
Tak, jeg kom et stykke videre nu. Er det korrekt at man ikke får nogen melding efter at have skrevet "ja" når der bliver spurgt om sagen skal lukkes? Kan man gå ud fra at der er en succes? :)

(fire) F:\GRF\Data\GEO\BC\Niv_Opgaver\2020\5D_KDI_KALVEHAVE>fire niv luk-sag 5D_KDI_KALVEHAVE
Er du sikker på at du vil lukke sagen 5D_KDI_KALVEHAVE? ja/[nej]ja

(fire) F:\GRF\Data\GEO\BC\Niv_Opgaver\2020\5D_KDI_KALVEHAVE>

**Reply by kbevers** [reply]
Ja, det er gået godt hvis ikke programmet brokker sig. Er det forvirrende?

**Reply by rebeccakol** [reply]
Måske lidt - det er i hvert fald anderledes når man har vænnet sig til at der ved de andre funktioner kommer et resultat på en eller anden måde frem som excel ark eller som opsummering.

**Reply by kbevers** [reply]
Så noget i den her stil?

```
(fire) C:\Temp\niv>fire niv luk-sag test
Er du sikker på at du vil lukke sagen test? ja/[nej]nej
Sag 500d2721-10ed-4fa7-9083-6d091dba162c for 'test' IKKE lukket!

(fire) C:\Temp\niv>fire niv luk-sag test
Er du sikker på at du vil lukke sagen test? ja/[nej]ja
Sag 500d2721-10ed-4fa7-9083-6d091dba162c for 'test' lukket!
```

Og hvis man prøver at lukke en sag der allerede er lukket for man denne besked:

```
(fire-dev) C:\Temp\niv>fire niv luk-sag test
Sag 500d2721-10ed-4fa7-9083-6d091dba162c for test er markeret inaktiv. Genåbn for at gå videre.
```

Vi mangler et værktøj til at genåbne sager med, det ryger på listen over ting der skal laves.

**Reply by rebeccakol** [reply]
Ja det er super!

**Reply by rebeccakol** [reply]
Hvordan genåbner man så en sag?

**Reply by kbevers** [reply]
Det gør man ikke endnu, der mangler et værktøj til det. Se https://github.com/Kortforsyningen/FIRE/issues/328. Jeg kan gøre det for dig hvis det er vigtigt, så skal jeg bare ha' sagens uuid.

**Reply by rebeccakol** [reply]
Det er okay - det var bare noget vi vendte på vores opstartsmøde her til morgen :)
